Index: src/LouiceConfiguration.as
===================================================================
*** src/LouiceConfiguration.as	(revision 13362)
--- src/LouiceConfiguration.as	(revision 13367)
***************
*** 29,35 ****
      import com.cenix.louice.shared.model.vos.MoleculeDesignPoolMember;
      import com.cenix.louice.shared.model.vos.MoleculeDesignPoolSetMember;
      import com.cenix.louice.shared.model.vos.MoleculeDesignSetMember;
-     import com.cenix.louice.shared.model.vos.MoleculeModificationMember;
      import com.cenix.louice.shared.model.vos.MoleculeTypeMember;
      import com.cenix.louice.shared.model.vos.OrganizationMember;
      import com.cenix.louice.shared.model.vos.PlateMember;
--- 29,34 ----
***************
*** 47,53 ****
      import com.cenix.louice.shared.model.vos.SubprojectMember;
      import com.cenix.louice.shared.model.vos.TagMember;
      import com.cenix.louice.shared.model.vos.TaggedRackPositionSetMember;
-     import com.cenix.louice.shared.model.vos.TargetSetMember;
      import com.cenix.louice.shared.model.vos.TubeMember;
      import com.cenix.louice.shared.model.vos.TubeRackMember;
      import com.cenix.louice.shared.model.vos.TubeRackSpecsMember;
--- 46,51 ----
***************
*** 155,161 ****
                  res.push( new SchemaModelEntry("molecule_design_pool", "http://schemata.cenix-bioscience.com/moleculedesignset", "MoleculeDesignPool",MoleculeDesignPoolMember) );
                  res.push( new SchemaModelEntry("molecule_design_pool_set", "http://schemata.cenix-bioscience.com/moleculedesignpoolset", "MoleculeDesignPoolSet",MoleculeDesignPoolSetMember, "/schemata/MoleculeDesignPoolSet.xsd") );
                  res.push( new SchemaModelEntry("molecule_design_set", "http://schemata.cenix-bioscience.com/moleculedesignset", "MoleculeDesignSet",MoleculeDesignSetMember, "/schemata/MoleculeDesignSet.xsd") );
-                 res.push( new SchemaModelEntry("molecule_modification", "http://schemata.cenix-bioscience.com/moleculemodification", "MoleculeModification",MoleculeModificationMember, "/schemata/MoleculeModification.xsd") );
                  res.push( new SchemaModelEntry("molecule_type", "http://schemata.cenix-bioscience.com/moleculetype", "MoleculeType",MoleculeTypeMember, "/schemata/MoleculeType.xsd") );
                  res.push( new SchemaModelEntry("organization", "http://schemata.cenix-bioscience.com/organization", "Organization",OrganizationMember, "/schemata/Organization.xsd") );
                  res.push( new SchemaModelEntry("project", "http://schemata.cenix-bioscience.com/project", "Project",ProjectMember, "/schemata/Project.xsd") );
--- 153,158 ----
***************
*** 173,179 ****
                  res.push( new SchemaModelEntry("subproject", "http://schemata.cenix-bioscience.com/subproject", "Subproject",SubprojectMember, "/schemata/Subproject.xsd") );
                  res.push( new SchemaModelEntry("tag", "http://schemata.cenix-bioscience.com/tag", "Tag",TagMember, "/schemata/Tag.xsd") );
                  res.push( new SchemaModelEntry("tagged_rack_position_set", "http://schemata.cenix-bioscience.com/taggedrackpositionset", "TaggedRackPositionSet",TaggedRackPositionSetMember, "/schemata/TaggedRackPositionSet.xsd") );
-                 res.push( new SchemaModelEntry("target_set", "http://schemata.cenix-bioscience.com/targetset", "TargetSet",TargetSetMember, "/schemata/TargetSet.xsd") );
                  res.push( new SchemaModelEntry("tube-rack", "http://schemata.cenix-bioscience.com/rack", "TubeRack",TubeRackMember) );
                  res.push( new SchemaModelEntry("tube_rack_specs", "http://schemata.cenix-bioscience.com/rackspecs", "TubeRackSpecs",TubeRackSpecsMember) );
                  res.push( new SchemaModelEntry("user", "http://schemata.cenix-bioscience.com/user", "User",UserMember, "/schemata/User.xsd") );
--- 170,175 ----
Index: src/com/cenix/louice/shared/model/vos/MoleculeModificationMember.as
===================================================================
*** src/com/cenix/louice/shared/model/vos/MoleculeModificationMember.as	(revision 13362)
--- src/com/cenix/louice/shared/model/vos/MoleculeModificationMember.as	(revision 13367)
***************
*** 1,14 ****
- package com.cenix.louice.shared.model.vos
- {
- 	import org.everest.flex.model.Member;
- 
-     public class MoleculeModificationMember extends Member
-     {
-         public var name:String;
- 
-         public function MoleculeModificationMember(title:String=null, selfLink:String=null)
-         {
-             super(title, selfLink);
-         }
-     }
- }
\ No newline at end of file
--- 0 ----
Index: src/com/cenix/louice/shared/model/vos/TargetSetMember.as
===================================================================
*** src/com/cenix/louice/shared/model/vos/TargetSetMember.as	(revision 13362)
--- src/com/cenix/louice/shared/model/vos/TargetSetMember.as	(revision 13367)
***************
*** 1,13 ****
- package com.cenix.louice.shared.model.vos
- {
- 	import org.everest.flex.model.Member;
- 
-     public class TargetSetMember extends Member
-     {
-         //TODO
-         public function TargetSetMember(title:String=null, selfLink:String=null)
-         {
-             super(title, selfLink);
-         }
-     }
- }
\ No newline at end of file
--- 0 ----
Index: src/com/cenix/louice/shared/model/vos/ExperimentMetaDataMember.as
===================================================================
*** src/com/cenix/louice/shared/model/vos/ExperimentMetaDataMember.as	(revision 13362)
--- src/com/cenix/louice/shared/model/vos/ExperimentMetaDataMember.as	(revision 13367)
***************
*** 8,14 ****
          public var label:String;
          public var subproject:SubprojectMember;
          public var molecule_design_pool_set:MoleculeDesignPoolSetMember;
-         public var target_set:TargetSetMember;
          public var experiment_design:ExperimentDesignMember;
          public var iso_request:IsoRequestMember;
          public var creation_date:Date;
--- 8,13 ----
Index: src/com/cenix/louice/shared/model/vos/ContainerMember.as
===================================================================
*** src/com/cenix/louice/shared/model/vos/ContainerMember.as	(revision 13362)
--- src/com/cenix/louice/shared/model/vos/ContainerMember.as	(revision 13367)
***************
*** 7,13 ****
      {
          public var rack:RackMember;
          public var rack_specs:RackSpecsMember;
!         public var position:String;
          public var sample_volume:Number;
          public var sample_molecule_design_pool_id:String;
          public var sample_molecules:MembersCollection;
--- 7,13 ----
      {
          public var rack:RackMember;
          public var rack_specs:RackSpecsMember;
!         public var position:RackPositionMember;
          public var sample_volume:Number;
          public var sample_molecule_design_pool_id:String;
          public var sample_molecules:MembersCollection;
Index: src/com/cenix/louice/modules/tubemember/ui/presenters/TubeMemberPresentationModel.as
===================================================================
*** src/com/cenix/louice/modules/tubemember/ui/presenters/TubeMemberPresentationModel.as	(revision 13362)
--- src/com/cenix/louice/modules/tubemember/ui/presenters/TubeMemberPresentationModel.as	(revision 13367)
***************
*** 2,7 ****
--- 2,8 ----
  {
      import com.cenix.louice.shared.model.vos.ContainerSpecsMember;
      import com.cenix.louice.shared.model.vos.ItemStatusMember;
+     import com.cenix.louice.shared.model.vos.RackPositionMember;
      import com.cenix.louice.shared.model.vos.RackSpecsMember;
      import com.cenix.louice.shared.model.vos.TubeMember;
      
***************
*** 37,43 ****
          }
  
          [Bindable(Event="memberChanged")]
!         public function get position():String
          {
              return TubeMember(_member).position;
          }
--- 38,44 ----
          }
  
          [Bindable(Event="memberChanged")]
!         public function get position():RackPositionMember
          {
              return TubeMember(_member).position;
          }
Index: src/com/cenix/louice/modules/experimentmetadatamember/ui/presenter/ExperimentMetaDataMemberPresentationModel.as
===================================================================
*** src/com/cenix/louice/modules/experimentmetadatamember/ui/presenter/ExperimentMetaDataMemberPresentationModel.as	(revision 13362)
--- src/com/cenix/louice/modules/experimentmetadatamember/ui/presenter/ExperimentMetaDataMemberPresentationModel.as	(revision 13367)
***************
*** 11,17 ****
      import com.cenix.louice.shared.model.vos.SubprojectMember;
      import com.cenix.louice.shared.model.vos.TagMember;
      import com.cenix.louice.shared.model.vos.TagPredicate;
-     import com.cenix.louice.shared.model.vos.TargetSetMember;
      
      import flash.events.Event;
      import flash.events.IEventDispatcher;
--- 11,16 ----
***************
*** 146,163 ****
          }
  
          [Bindable(Event="memberChanged")]
-         public function get target_set():TargetSetMember
-         {
-             return ExperimentMetaDataMember(_member).target_set;
-         }
- 
-         public function set target_set(target_set:TargetSetMember):void
-         {
-             ExperimentMetaDataMember(_member).target_set = target_set;
-             dispatchEvent(new Event(MemberEvent.MEMBER_CHANGED));
-         }
- 
-         [Bindable(Event="memberChanged")]
          public function get experimentDesign():ExperimentDesignMember
          {
              return ExperimentMetaDataMember(_member).experiment_design;
--- 145,150 ----
